website/config.toml

```diff
@@ -199,7 +199,7 @@ url_latest_version = "https://sdk.operatorframework.io"
 
 [[params.versions]]
   version = "v1.20"
-  url = "https://v1-20-x.sdk.operatorframework.io"
+  url = "https://github.com/operator-framework/operator-sdk/tree/v1.20.x/website/content/en/docs"
```

Comment:
I think this should be reverted back. This change is made only with the release PR.

Reply:
This is needed since `v1-20-x` is no longer a subdomain, without this change the docs GHA fails